config.json

```diff
@@ -6,7 +6,7 @@
   "matching_results_folder": "test-data/envs/small/results",
   "output_folder": "test-data/envs/small/output",
   "entity_service_url": "http://localhost:8851/api/v1",
-  "matching_threshold": 0.75,
+  "matching_threshold": [0.75,0.75,0.75,0.75],
```

Comment:
This is super trivial but because the order matters, could you move this key up to just below the "projects" line? Maybe even add spaces to the matching thresholds to make it more obvious that there's a relationship between them and see how they line up. Ex:
```json
"projects": ["name-sex-dob-phone", "name-sex-dob-zip", "name-sex-dob-parents", "name-sex-dob-addr"],
"matching_threshold": [0.75,               0.75,               0.75,                 0.75],

```

Reply:
Good idea. Will do